api/tests/python/unit/test_setup_utils/build_nr.py

```diff
@@ -68,12 +69,12 @@ def build_nr(nr_state, data=None, test_names=None, generate_id_seq=None):
 
 def build_draft(data=None, test_names=None, generate_id_seq=None):
     try:
-        nr = RequestDAO()
+        nr = nr_json()
```

Comment:
Hey andre, what's the purpose of changing this method? The problem now is that all the other 'build_..' methods for the other states will create a different response type, which will make using *build_nr* very confusing later. Also, *build_nr* is a lot more useful for most tests if it returns the NR model and not just the json as you are doing here

Reply:
Yeah, ok... I don't think it's gonna work the way it was supposed to work anymore because of changes in the model but I'm gonna revert this and create my own NR json without using this function just to preserve it.

Reply:
okay sounds good